assets/queries/cloudFormation/emr_security_configuration_encryptions_enabled/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "emr_security_configuration_encryptions_enabled",
+  "queryName": "EMR Security Configuration Encryptions Enabled",
+  "severity": "MEDIUM",
+  "category": null,
+  "descriptionText": "EMR SecurityConfiguration should enable and properly configure encryption at rest and in transit.",
+  "descriptionUrl": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-emr-securityconfiguration.html"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline